tests/ui/job-view/SecondaryNavBar_test.jsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { fetchMock } from 'fetch-mock';
+import { Provider } from 'react-redux';
+import { render, cleanup, waitForElement } from '@testing-library/react';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+
+import { replaceLocation, setUrlParam } from '../../../ui/helpers/location';
+import FilterModel from '../../../ui/models/filter';
+import { PinnedJobs } from '../../../ui/job-view/context/PinnedJobs';
+import SecondaryNavBar from '../../../ui/job-view/headerbars/SecondaryNavBar';
+import { initialState } from '../../../ui/job-view/redux/stores/pushes';
+
+const mockStore = configureMockStore([thunk]);
+const repoName = 'autoland';
+const repos = [
+  {
```

Comment:
I wonder if we should start putting mock data in separate files to cut down on boilerplate. I might be able to use this for some perfherder tests. Also, it looks like you'd be able to use this for the `currentRepo` in the `Pushlist_test` instead of hardwiring it there.

Reply:
Yeah, we do for a few.  But it's true, we should do that here as well.